active_learning_multiannotator/utils/active_learning.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
```

Comment:
would be nice to have one-liner comment at top describing the purpose of this file